src/backend/distributed/commands/sequence.c

```diff
@@ -213,6 +213,43 @@ ExtractDefaultColumnsAndOwnedSequences(Oid relationId, List **columnNameList,
 }
 
 
+/*
+ * RelationGetDefaultNextValAttrs returns a list of AttrNumbers for the
+ * columns that default to nextval().
+ */
+List *
+RelationGetDefaultNextValAttrs(Oid relationId)
+{
+	List *defaultNextValAttrs = NIL;
+
+	Relation relation = RelationIdGetRelation(relationId);
+
+	TupleDesc relTupDesc = RelationGetDescr(relation);
+	TupleConstr *relTupConstraints = relTupDesc->constr;
+	if (relTupConstraints == NULL)
+	{
+		RelationClose(relation);
+		return NIL;
+	}
+
+	for (uint16 defExprIndex = 0; defExprIndex < relTupConstraints->num_defval;
```

Comment:
can we use `build_column_default `?

Reply:
instead dropped that function altogether